package.json

```diff
@@ -4,19 +4,32 @@
   "description": "A smart request-making library that makes sure your Javascript client is a good citizen of your distributed system.",
   "main": "dist/index.js",
   "scripts": {
-    "clean": "rm -rf ./dist",
-    "build": "npm run clean && tsc",
+    "//": "Typescript Compilation",
```

Comment:
In general, I'm not a huge fan of docs like this. The scripts are simple enough to be self explanatory – more docs should go in the README.md